### PR TITLE
concat fix

### LINKED ISSUE #71: old style concatenation warning
solved a MIGP model with Pajarito (Gurobi and KNITRO)
```
WARNING: [a,b] concatenation is deprecated; use [a;b] instead
 in depwarn at deprecated.jl:73
 in oldstyle_vcat_warning at /Applications/Julia-0.4.3.app/Contents/Resources/julia/lib/julia/sys.dylib
 in vect at abstractarray.jl:38
 in optimize! at /Users/chris/.julia/v0.4/Pajarito/src/nonlinear.jl:669
 in solvenlp at /Users/chris/.julia/v0.4/JuMP/src/nlp.jl:1219
 in solve at /Users/chris/.julia/v0.4/JuMP/src/solvers.jl:135
```

### PR DESCRIPTION
**Pull request by emreyamangil**
Closes #71

### CODE REVIEW

**Comment by mlubin** [comment]
:+1:

File: src/nonlinear.jl [hunk 1]
```diff
@@ -665,8 +665,10 @@ function MathProgBase.optimize!(m::PajaritoModel)
             # WARMSTART MIP FROM UPPER BOUND
             if !any(isnan,m.solution) && !isempty(m.solution)
                 if applicable(MathProgBase.setwarmstart!, getInternalModel(mip_model), m.solution)
+                    mip_warmstart = copy(m.solution)
+                    push!(mip_warmstart, m.objval)
                     # Extend solution with the objective variable
-                    MathProgBase.setwarmstart!(getInternalModel(mip_model), [m.solution, m.objval])
+                    MathProgBase.setwarmstart!(getInternalModel(mip_model), mip_warmstart)
```
Comment (mlubin) [inline src/nonlinear.jl]: Just use ``[m.solution; m.objval]`` and save 2 lines

Comment (emreyamangil) [inline src/nonlinear.jl]: I was thinking what if the solver returns a column vector vs row vector, can it cause an issue?

Comment (mlubin) [inline src/nonlinear.jl]: Solvers always return plain 1d vectors

Comment (emreyamangil) [inline src/nonlinear.jl]: You got it! Reverting back to old version!

**Pull request merged**